CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
 - Adds new command ``quota-show``.
-- Adds possibility to filter by workflow status and search by workflow name to ``list`` command.
+- Adds new command ``info`` that retrieves general information about the cluster, including the list all the available workspace paths.
+- Adds possibility to filter by workflow status and search by workflow name to ``list`` command via ``--filter`` option.
+- Adds possibility to filter and display all the runs of a given workflow to the ``list`` command via ``-w`` option.
+- Adds possibility to filter by file name, size and last modified values to ``ls`` command via ``--filter`` option.
+- Adds possibility to filter by file name and size to ``du`` command via ``--filter`` option.
 - Adds job start and finish times to ``logs`` command output.
-- Adds possibility to filter by file name, size and last modified values to ``ls`` command.
-- Adds new command ``info`` that retrieves general information about the cluster.
 - Adds new ``--server-capabilities`` flag to ``validate`` command.
-- Adds the possibility to filter and display all the runs of a given workflow to the ``list`` command.
+- Adds support for ``outputs.directories`` in ``reana.yaml``.
+- Adds support for Snakemake workflow submission and validation.
+- Adds support for ``cwltool`` version ``3.1.20210628163208``.
```

Comment:
This should be "changes", i.e. not a new thing, but an upgrade from CWL version N1 to CWL version N2.

Reply:
Actually, it should be "Upgrades", I don't really understand this policy of just keeping "Adds/Changes/Fixes/Deletes".

Reply:
It is just a way to represent the traditional "New features / Changed behaviour / Bug fixes / Deprecated" division of news items in the release notes, using common language structure rather than introducing one more list level indentation in 
the RST used elsewhere e.g. in https://www.sphinx-doc.org/en/master/changes.html#release-4-0-0-released-may-09-2021